cmd/libs/go2idl/set-gen/sample_output/k8s.io/kubernetes/pkg/util/sets/byte.go

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
To be honest - I think that we want to remove the whole second commit form this PR.
You already have third commit, where you generate something useful, so I vote for removing the whole second commit.